force-app/main/default/layouts/Contact-Contact %28Sales%29 Layout.layout-meta.xml

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I am not sure if adding or modifying the standard layouts is a good idea! If someone installs apex-recipes in production as an unlocked package (or deploying it), there will be overrides.